### PR TITLE
add a button

### PR DESCRIPTION
**Pull request by mergeflydemo**
Add a new button to the page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -19,6 +21,7 @@ function App() {
           Learn React
         </a>
       </header>
+      <button onClick={()=>setCount(count + 1)}>I've been clicked {count} times!</button>
```
Comment (avisprince) [inline src/App.js]: It looks like you didnt add any tests!!

Comment (mergeflydemo) [reply src/App.js]: I will add them right away!!

Comment (avisprince) [reply src/App.js]: hi

Comment (avisprince) [reply src/App.js]: hi

**Pull request closed without merging**